scripts/run_smoke_test.sh

```diff
@@ -99,7 +99,7 @@ run_smoke_test() {
     kubectl run nginx --image=nginx --port=80
     kubectl expose pod nginx
     kubectl run test --restart=Never --image=elotl/debug --command -- /bin/sh -c "$curlcmd"
-    timeout 300s bash -c "$waitcmd"
+    timeout 360s bash -c "$waitcmd"
```

Comment:
If 300 seconds wasn't enough and made the test flaky, I would double it to 600 seconds to ensure the flakes won't come back to haunt us.

Reply:
I set it to 420s. It should be more than sufficient; if it is exceeded there's clearly something wrong. I don't want to bump it to 10 mins as those builds are already ~20-25 mins long